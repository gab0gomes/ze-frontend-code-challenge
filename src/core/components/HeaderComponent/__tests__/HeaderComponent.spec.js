import { shallowMount, createLocalVue } from '@vue/test-utils'

import Header from '../index'

const localVue = createLocalVue()

describe('HeaderComponent', () => {
  it('render correctly', async () => {
    const wrapper = shallowMount(Header, {
      localVue
    })

    expect(wrapper.element).toMatchSnapshot()

    wrapper.destroy()
  })
})
