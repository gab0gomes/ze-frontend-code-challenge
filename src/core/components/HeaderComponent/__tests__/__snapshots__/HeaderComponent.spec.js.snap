// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderComponent render correctly 1`] = `
<header>
  <nav>
    <logo-stub
      class="logo"
    />
     
    <span>
      Frontend Challenge
    </span>
  </nav>
</header>
`;
